ssh_check/datadog_checks/ssh_check/ssh_check.py

```diff
@@ -76,9 +76,9 @@ def check(self, instance):
         if conf.private_key_file is not None:
             try:
                 if conf.private_key_type == 'ecdsa':
-                    private_key = paramiko.ECDSAKey.from_private_key_file(conf.private_key_file)
+                    private_key = paramiko.ECDSAKey.from_private_key_file(conf.private_key_file, password=conf.password)
```

Comment:
Would be great to have an integration/e2e test for this case. Let me know if you need help for this.

Reply:
Yes, please. How would I go about creating an e2e test case?

Reply:
like this: https://github.com/DataDog/integrations-core/pull/7655